pytest_reportportal/service.py

```diff
@@ -449,6 +449,19 @@ def _get_item_dirs(item):
 
         return dir_list
 
+    def _get_launch_tags(self, ini_tags):
+        """Generate launch tags in the format supported by the client.
+
+        :param list ini_tags: List for tags from the pytest.ini file
+        """
+        tags = ini_tags if ini_tags else []
```

Comment:
```suggestion
        tags = ini_tags or []
```

Reply:
This is a pattern a friend of mine spotted in the Python core source code. Useful shorthand and worth using if you think it is readable enough.